### DISCUSSION TITLE
[SUGGESTION] Cuboid Claims

### DISCUSSION THREAD

**Discussion by iSyriux2** [GriefPrevention/GriefPrevention#1460] [Ideas]
So I've had this idea since 2019. I had this floating desert base in a towny server and I had a plan for vertical apartments but then I found out that it took the entire chunk, so if I had a public facility under someone's private plot, they could come and grief it.
Because of that I quit the server because just 2d claiming was plain and boring, and I had been playing for a few months already.
My dream of a towering dystopian floating desert city was crushed because cuboid claims didn't exist yet. Imagine how cool it would be to have like a street with tall buildings and each floor was owned by a different person and you could navigate around the city vertically, like kowloon walled city
![image](https://user-images.githubusercontent.com/62969957/125690552-586cb8bf-c264-4c9b-8191-5e943926504a.png)
This sponge version has cuboid claims so you could take the idea directly from them https://ore.spongepowered.org/blood/GriefPrevention

**Comment by RoboMWM** [comment]
Duplicate of #269
